### PR TITLE
[ffmpeg] Update to 4.3.1

### LINKED ISSUE #13392: [ffmpeg] update to 4.3.1
Library name:
`ffmpeg`

New version number:
`4.3.1`

Other information that may be useful (release notes, etc...)
`https://ffmpeg.org/download.html#release_4.3`
```
4.3.1 was released on 2020-07-11. It is the latest stable FFmpeg release from the 4.3 release branch, which was cut from master on 2020-06-08.

It includes the following library versions:

libavutil      56. 51.100
libavcodec     58. 91.100
libavformat    58. 45.100
libavdevice    58. 10.100
libavfilter     7. 85.100
libswscale      5.  7.100
libswresample   3.  7.100
libpostproc    55.  7.100
```

### PR DESCRIPTION
**Pull request by Sibras**
Also update ffnvcodec to 10.0.26.0.

Closes #13392

### CODE REVIEW

**Comment by Sibras** [comment]
Add an extra commit that fixes a regression when vorbis was updated to 1.3.7. This fixes detection by ffmpeg on linux.

Ive tested with all ports enabled on x64-windows, x64-windows-static and x64-linux and everything works correctly (except x264 as that currently doesnt build on windows and needs to be fixed).

**Comment by LilyWangL** [comment]
Thanks for your PR. Now `aubio:x64-windows-static` build failed with the following error, it build depends on `ffmpeg`.
```
LINK: command "C:\PROGRA~2\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.27.29110\bin\Hostx64\x64\link.exe CMakeFiles\aubionotes.dir\examples\aubionotes.c.obj CMakeFiles\aubionotes.dir\examples\utils.c.obj CMakeFiles\aubionotes.dir\examples\jackio.c.obj /out:aubionotes.exe /implib:aubionotes.lib /pdb:aubionotes.pdb /version:0.0 /machine:x64 /nologo /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF /subsystem:console aubio.lib D:\installed\x64-windows-static\lib\sndfile.lib D:\installed\x64-windows-static\lib\vorbisenc.lib D:\installed\x64-windows-static\lib\vorbis.lib D:\installed\x64-windows-static\lib\FLAC.lib D:\installed\x64-windows-static\lib\ogg.lib D:\installed\x64-windows-static\lib\opus.lib D:\installed\x64-windows-static\lib\avcodec.lib D:\installed\x64-windows-static\lib\avutil.lib D:\installed\x64-windows-static\lib\avdevice.lib D:\installed\x64-windows-static\lib\avfilter.lib D:\installed\x64-windows-static\lib\avformat.lib D:\installed\x64-windows-static\lib\swresample.lib D:\installed\x64-windows-static\lib\bz2.lib D:\installed\x64-windows-static\lib\lzma.lib ws2_32.lib Secur32.lib bcrypt.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:aubionotes.exe.manifest" failed (exit code 1120) with the following output:
avcodec.lib(mfenc.o) : error LNK2019: unresolved external symbol MFCreateMediaType referenced in function mf_choose_output_type
avcodec.lib(mfenc.o) : error LNK2001: unresolved external symbol IID_ICodecAPI
avcodec.lib(mfenc.o) : error LNK2001: unresolved external symbol IID_IMFMediaEventGenerator
avcodec.lib(mf_utils.o) : error LNK2019: unresolved external symbol MFStartup referenced in function ff_instantiate_mf
avcodec.lib(mf_utils.o) : error LNK2019: unresolved external symbol MFShutdown referenced in function ff_free_mf
avcodec.lib(mf_utils.o) : error LNK2019: unresolved external symbol MFCreateAlignedMemoryBuffer referenced in function ff_create_memory_sample
avcodec.lib(mf_utils.o) : error LNK2019: unresolved external symbol MFCreateSample referenced in function ff_create_memory_sample
avcodec.lib(mf_utils.o) : error LNK2001: unresolved external symbol IID_IMFTransform
aubionotes.exe : fatal error LNK1120: 8 unresolved externals
```

**Comment by Sibras** [comment]
aubio seems to be using a custom windows only cmake file. This just needs to be updated to add "mfplat.lib mfuuid.lib strmiids.lib" to target_link_libraries. The way the aubio cmake file is currently written is not very portable (hence why it broke) so I havnt made a patch with the above fix in case someone wants to do a proper fixup.

**Comment by Sibras** [comment]
I updated aubio to use FindFFmpeg which fixes the issue in a more portable way.

**Review by LilyWangL — APPROVED** [review]

**Comment by sdcb** [comment]
I confirmed this is working for me in x64-windows/x64-windows-static, thanks @Sibras .

File: ports/aubio/CMakeLists.txt [hunk 1]
```diff
@@ -24,12 +24,7 @@ set(CMAKE_DEBUG_POSTFIX d)
 option(BUILD_TOOLS "Build and install tools" ON)
```
Comment (ras0219-msft) [inline ports/aubio/CMakeLists.txt:24]: This change requires an update to `aubio`'s Port-Version.

**Review by ras0219-msft — APPROVED** [review]
LGTM after bumping `aubio`'s version.

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**